main.py

```diff
@@ -1,10 +1,52 @@
+from django.db.models import ForeignKey  # noqa: F401
 import init_django_orm  # noqa: F401
-
+import json
+import datetime
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players_file:
+        players = json.load(players_file)
+
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Player.objects.all().delete()
+    Guild.objects.all().delete()
```

Comment:
Deleting all objects from the database tables at the start of the script can lead to data loss. Ensure this is the intended behavior, or consider implementing a safer data handling strategy.